ext/uriparser_ext/extconf.rb

```diff
@@ -3,9 +3,6 @@
 require 'mkmf'
 require 'rbconfig'
 
-unless have_library('uriparser')
-  error 'You need to install uriparser lib'
-  exit
-end
+abort "-----\nERROR: You need to install uriparser lib.\n-----" unless have_library('uriparser')
```

Comment:
What about:

```ruby
have_library('uriparser') || abort("-----\nERROR: You need to install uriparser lib.\n-----")
```

I guess it would make the code intent clearer.

Reply:
Sure, pushing it now 👍

Reply:
Thanks!